src/views/Setup.vue

```diff
@@ -45,7 +45,15 @@ function handleSubmit() {
           </a>
         </UiButton>
         <UiButton
-          :disabled="!id.includes('.eth') && !id.includes('.xyz')"
+          :disabled="
+            !id.includes('.eth') &&
```

Comment:
Is the list of supported domains referenced somewhere? Would be better to make an array list on the script like
```
const tlds = ['.eth', '.xyz', '.com', ...];
```
And make a function instead of writing long statement in the view.

For the function you could use "endsWith" instead of includes so it fail is someone type "abc.eth.def"

Reply:
Perfect makes sense 👍🏻

Reply:
@bonustrack Can you please review again 🙇🏻